src/main/java/net/dv8tion/jda/api/interactions/commands/CommandInteraction.java

```diff
@@ -200,4 +200,54 @@ default OptionMapping getOption(@Nonnull String name)
         List<OptionMapping> options = getOptionsByName(name);
         return options.isEmpty() ? null : options.get(0);
     }
+
+    /**
+     * Gets the slash command String for this slash command.
+     * <br>This is similar to the String you see when clicking the interaction name in the client.
+     *
+     * <p>Example return for an echo command: {@code /say echo phrase: Say this}
+     *
+     * @return The command String for this slash command
+     */
+    @Nonnull
+    default String getCommandString()
+    {
+        //Get text like the text that appears when you hover over the interaction in discord
+        StringBuilder builder = new StringBuilder();
+        builder.append("/").append(getName());
+        if (getSubcommandGroup() != null)
+            builder.append(" ").append(getSubcommandGroup());
+        if (getSubcommandName() != null)
+            builder.append(" ").append(getSubcommandName());
+        builder.append(" ");
+        //build options (formatted appropriately)
+        for (OptionMapping o : getOptions())
+        {
+            builder.append(o.getName()).append(":");
+            switch (o.getType())
+            {
+            case CHANNEL:
+                builder.append(" ").append("#").append(o.getAsGuildChannel().getName());
+                break;
+            case USER:
+                builder.append(" ").append("@").append(o.getAsUser().getName());
+                break;
+            case ROLE:
+                builder.append(" ").append("@").append(o.getAsRole().getName());
+                break;
+            case MENTIONABLE: //client only allows user or role mentionable
+                if (o instanceof Role)
+                    builder.append(" ").append("@").append(o.getAsRole().getName());
+                else if (o instanceof User)
+                    builder.append(" ").append("@").append(o.getAsUser().getName());
+                else
+                    builder.append(" ").append("@").append(o.getAsMentionable().getIdLong());
+                break;
+            default:
+                builder.append(" ").append(o.getAsString());
+                break;
+            }
+        }
+        return builder.toString().trim();
```

Comment:
This is no longer needed as we are no longer are appending `" "` after each entitiy

```suggestion
        return builder.toString();
```

Reply:
implemented changes